scripts/test/runtests.ps1

```diff
@@ -5,65 +5,79 @@
 
 . "$PSScriptRoot\..\common\_common.ps1"
 
-$TestBinRoot = "$RepoRoot\artifacts\tests"
+$failCount = 0
+
+# Capture PATH for later
+$StartPath = $env:PATH
```

Comment:
Why force this file only to work for stage2 binaries? Can we do this in the calling script?